src/configuration.ts

```diff
@@ -19,6 +19,13 @@ import * as fs from "fs";
 import * as path from "path";
 import * as findup from "findup-sync";
 
+import {arrayify, objectify} from "./utils";
+
+export interface IConfigFile {
```

Comment:
anything exported _is_ part of the public API; let's just assume people will use anything they can find in our generated typings